### DISCUSSION TITLE
ECS deployment process after Docker Compose retirement?

### DISCUSSION THREAD

**Discussion by dagsir** [dagster-io/dagster#17817] [Q&A]
Hey All!

I am looking through the Dagster ECS deployment documentation here: <https://docs.dagster.io/deployment/guides/aws#deploying-in-ecs> which lead me to the Docker site (<https://docs.docker.com/cloud/ecs-integration/>) and I noticed that the Docker Compose integration for ECS is to be retired end of the month (Nov 2023). Is there anyone who can guide me on what the ECS deployment process will look like after this retirement or what alternative methods exist for ECS deployment without using Docker Compose and what that looks like as well.

And please, anyone correct me if I'm not understanding something fully here...

Thanks in advance!

---

*The question was originally asked in [Dagster Slack](https://dagster.slack.com/archives/C014UDS8LAV/p1699453156535399?thread_ts=1699453156.535399&cid=C014UDS8LAV).*

**Comment by amouly** [comment]
We use CDK to configure and deploy Dagster infrastructure into AWS ECS.

Maybe we just need to provide better documentation on this setup.

**Reply by benjaminifaught** [reply]
Thanks for the reply! How does this process differ from the Docker Compose example in the Dagster deployment docs? Do you have any examples of what that looks like?

**Reply by amouly** [reply]
I don't have examples, but it's easy to setup.

Look into AWS CDK examples, it's simply building all the AWS components with code, and deploy them.

**Reply by benjaminifaught** [reply]
Does the AWS CDK allow for docker compose? If that's the case I might be able to still follow the documentation on the Dagster site...

**Reply by benjaminifaught** [reply]
@amouly Would you be able/willing to put together some basic documentation for how you do this via AWS CDK? My team is looking into different deployment options currently. Thanks!

**Reply by srujithpoondla03** [reply]
It would be really helpful to have this documentation or even an example CDK project should help.
